vaadin-cdi/src/test/java/com/vaadin/cdi/internal/MyBean.java

```diff
@@ -10,7 +9,6 @@
     private final int id = counter++;
 
     public MyBean() {
-        System.out.println(UI.getCurrent());
```

Comment:
I've found this: https://java.net/jira/browse/GLASSFISH-21242.
According to this [changelog](https://payara.gitbooks.io/payara-server/content/v/162/release-notes/release-notes-144.html) payara fixed it. I can switch to payara if you think it worth.